packages/clarity-js/types/core.d.ts

```diff
@@ -102,6 +102,7 @@ export interface Config {
     regions?: Regions;
     metrics?: Metrics;
     cookies?: string[];
+    server?: string;
```

Comment:
What if this is empty/undefined?

Reply:
Then that's fine - it will be handled like how we handled empty upload (or upload callback) scenario.